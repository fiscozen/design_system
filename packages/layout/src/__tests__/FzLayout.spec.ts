import { describe, it, vi } from 'vitest'
import { mount } from '@vue/test-utils'
import { FzLayout } from '..'

describe.concurrent('FzLayout', () => {
  it('matches snaphost', async ({ expect }) => {
    const wrapper = mount(FzLayout, {
      props: {},
      slots: {}
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
