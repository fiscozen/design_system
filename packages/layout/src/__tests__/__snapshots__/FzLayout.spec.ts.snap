// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FzLayout > matches snaphost for multipleRows 1`] = `
"<div class="fz-layout w-dvw h-dvh grid gap-10 p-16 grid-rows-auto">
  <!--v-if-->
  <!--v-if-->
</div>"
`;

exports[`FzLayout > matches snapshot for leftShoulder 1`] = `
"<div class="fz-layout w-dvw h-dvh grid gap-10 p-16 grid-rows-[60px_1fr] grid-cols-1 lg:grid-rows-1 lg:grid-cols-[260px_1fr]">
  <!--v-if-->
  <!--v-if-->
</div>"
`;

exports[`FzLayout > matches snapshot for leftShoulderNavbar 1`] = `
"<div class="fz-layout w-dvw h-dvh grid gap-10 p-16 grid-rows-[40px_160px_60px_1fr] grid-cols-1 lg:grid-rows-[40px_160px_1fr] lg:grid-cols-[260px_1fr]">
  <!--v-if-->
  <div class="col-span-2"></div>
  <div class="col-span-2"></div>
  <div class="col-span-2 lg:col-span-1"></div>
  <div class="col-span-2 lg:col-span-1"></div>
  <!--v-if-->
</div>"
`;

exports[`FzLayout > matches snapshot for rightShoulderNavbar 1`] = `
"<div class="fz-layout w-dvw h-dvh grid gap-10 p-16 grid-rows-[40px_160px_60px_1fr] grid-cols-1 lg:grid-rows-[40px_160px_1fr] lg:grid-cols-[1fr_260px]">
  <!--v-if-->
  <div class="col-span-2"></div>
  <div class="col-span-2"></div>
  <div class="col-span-2 lg:col-span-1"></div>
  <div class="col-span-2 lg:col-span-1"></div>
</div>"
`;

exports[`FzLayout > matches snapshot for squares 1`] = `
"<div class="fz-layout w-dvw h-dvh grid gap-10 p-16 grid-cols-1 grid-rows-auto sm:grid-cols-2 sm:grid-rows-auto md:grid-cols-3 md:grid-rows-auto auto-rows-[332px]">
  <!--v-if-->
  <!--v-if-->
</div>"
`;

exports[`FzLayout > matches snapshot for twoColumns  1`] = `
"<div class="fz-layout w-dvw h-dvh grid gap-10 p-16 grid-cols-1 grid-rows-[60px_1fr] lg:grid-rows-1 lg:grid-cols-2">
  <!--v-if-->
  <!--v-if-->
</div>"
`;
